modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -15,16 +15,10 @@ class ClaimsAndAppealsController < ApplicationController
       include IgnoreNotFound
       before_action(only: %i[get_appeal]) { authorize :appeals, :access? }
 
-      before_action(only: %i[get_claim request_decision]) do
-        if Flipper.enabled?(:mobile_lighthouse_claims, @current_user)
-          authorize :lighthouse, :access?
-        else
-          authorize :evss, :access?
-        end
-      end
-
-      before_action(only: %i[upload_document upload_multi_image_document]) do
-        if Flipper.enabled?(:mobile_lighthouse_document_upload, @current_user)
+      before_action(only: %i[get_claim request_decision upload_document upload_multi_image_document]) do
```

Comment:
In the PR description, you say that request decision is not ready. By changing this before action to include request_decision while also using lighthouse for all three of these feature flags, are you (partly) defeating the purpose of separating the feature flag? The actual request decision code is using this flag, but if the user has any of these flags, they're going to be authorized against lighthouse. That's probably not a huge issue because the policies are similar, but it still doesn't seem exactly correct to me. I think this is why there were multiple before actions with different feature flags.

On something of a side note, does this in any way impact `modules/mobile/app/models/mobile/v0/user_accessible_services.rb`? In that, all of claims is behind the `mobile_lighthouse_claims` flag. Should the request decision flag be represented anywhere? it's probably hard to know without knowing how this works on the front end.

Reply:
hm this is a good call out. I think in the claims controller for now we should probably have a before action for specific methods that relate to each flag. In this case, 1 for get_claim, 1 for request_decision, 1 for upload_document and upload_multi_image_document. 

As for the authed services I think it's fine to have key off of the LH claims flag. I don't think there's any reality where we have claims off, but the other two on. Worst case some some people might receive an evss access failure when actually making the RD or doc upload calls when authed services lets them through based on LH access success, but I don't think that's a very likely scenario